.github/config/cargo-deny.toml

```diff
@@ -21,6 +20,14 @@ skip-tree = [
     { name = "bolero" },
     { name = "criterion" },
     { name = "insta" },
+
+    # Ignore duplicate dependencies in private s2n-quic crates
+    { name = "s2n-quic-bench" },
+    { name = "s2n-quic-events" },
+    { name = "s2n-quic-h3" },
+    { name = "s2n-quic-qns" },
+    { name = "s2n-quic-sim" },
+    { name = "s2n" },
```

Comment:
This isn't a crate

```suggestion
```

Reply:
Sorry... Fixed. 68c496b2b5cb5989c0cfc1ff217dbe15a64623b4